bcda/web/api.go

```diff
@@ -413,7 +413,8 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 	switch job.Status {
 
 	case "Failed":
-		responseutils.WriteError(&fhirmodels.OperationOutcome{}, w, http.StatusInternalServerError)
```

Comment:
Can we add a test to verify we get the expected operational outcome when we encounter a job failure?